lib/active_fedora/associations/belongs_to_association.rb

```diff
@@ -36,16 +36,6 @@ def remove_keys
           owner[reflection.foreign_key] = nil
         end
 
-        # Constructs a query that checks solr for the correct id & class_name combination
-        def construct_query(ids)
```

Comment:
It was added here: https://github.com/projecthydra/active_fedora/commit/8ac538eb4442bde23e561735e00d28ff50d55744